bridge/client/app/project-board/project-board.component.ts

```diff
@@ -48,6 +51,8 @@ export class ProjectBoardComponent implements OnInit, OnDestroy {
   public eventTypes: string[] = [];
   public filterEventTypes: string[] = [];
 
+  public openApprovals: Event[] = [];
```

Comment:
I still think it is confusing because their is also the openApprovals$ variable in line 29.
I recommend to use different names, more specific ones.

Reply:
Oh sorry, the variable on line 54 isn't used at all, because there is the openApprovals$.
I removed the unused variable on line 54.